src_c/geometry.c

```diff
@@ -74,6 +176,7 @@ geometry_regular_polygon(PyObject *_null, PyObject *const *args,
 static PyMethodDef _pg_module_methods[] = {
     {"regular_polygon", (PyCFunction)geometry_regular_polygon, METH_FASTCALL,
      NULL},
+    {"raycast", (PyCFunction)pg_raycast, METH_VARARGS | METH_KEYWORDS, NULL},
```

Comment:
METH_FASTCALL here

Reply:
I tried this before but it kept segfaulting earlier, if I remove the args and kwargs parser it might work. Idk.